### PR TITLE
terminalizer 0.9 (new formula)

### PR DESCRIPTION
**Pull request by b-reich**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by b-reich** [comment]
```
terminalizer:
  * Binaries built for a non-native architecture were installed into terminalizer's prefix.
    The offending files are:
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-arm64/node.abi64.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-arm64/node.abi67.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-arm64/node.abi72.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-arm64/node.abi79.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-arm64/node.abi83.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi102.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi108.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi64.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi67.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi72.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi79.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi83.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi88.node	(x86_64)
      /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/node-pty-prebuilt-multiarch/prebuilds/darwin-x64/node.abi93.node	(x86_64)
Error: 1 problem in 1 formula detected
```
Not sure if this okay or not. When not I create an Bug on their Github.

File: Formula/terminalizer.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  homepage "https://www.terminalizer.com/"
+  url "https://registry.npmjs.org/terminalizer/-/terminalizer-0.9.0.tgz"
+  sha256 "512ec9dc30a5d0a5471223352fb14094222e684f72c5a9c23c196a4d90206fb0"
+  license "MIT"
+  head "https://github.com/faressoft/terminalizer.git", branch: "master"
+
+  livecheck do
+    url :stable
+  end
+
```
Comment (chenrui333) [inline Formula/terminalizer.rb:14]:
```suggestion
```

**Comment by chenrui333** [comment]
after fixing the linkage issue, now run into 

```
  ln -s ../Cellar/terminalizer/0.9.0/bin/terminalizer terminalizer
  Error: Failed changing dylib ID of /opt/homebrew/Cellar/terminalizer/0.9.0/libexec/lib/node_modules/terminalizer/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
    from @rpath/Electron Framework.framework/Electron Framework
      to /opt/homebrew/opt/terminalizer/libexec/lib/node_modules/terminalizer/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
```

**Comment by b-reich** [comment]
> Consider it has bundled electron, I might think this is more suitable to be a cask.

How should the cask look like?

**Pull request closed without merging**